stack_overflow/src/view/answer/AnswersInput.js

```diff
@@ -0,0 +1,14 @@
+
+import React from "react";
+
+const AnswersInput = ({ text, currentQuestion, onChange, onCreate }) => (
+    <div>
+        <label> Text: </label>
+        <input value={text}
+            onChange={e => onChange("text", e.target.value)} />
+        <br />
+        <button onClick={() => onCreate(currentQuestion)}> Create!</button>
```

Comment:
This is a little hacky, you could pass the current question to the event handler inside the Smart Component instead.